package.json

```diff
@@ -8,14 +8,17 @@
     "lint": "eslint . --ext .js",
     "test": "npm run lint && npm run test:server && npm run test:browser",
     "test:browser": "web-test-runner --files \"./test/browser/**/*.test.js\" --node-resolve --playwright",
-    "test:server": "mocha ./test/server/**/*.test.js"
+    "test:server": "mocha ./test/server/**/*.test.js",
+    "test:vdiff": "web-test-runner --config ./test/vdiff/wtr-vdiff.config.js --group vdiff",
```

Comment:
This can't run in actual CI yet, and won't be able to until we have the action. But for now it's super helpful to run locally to dev the plugin.